### ISSUE TITLE
chunksize=True raises ValueError

### ISSUE THREAD

**Issue by czagoni** [awslabs/aws-data-wrangler#458]
**Describe the bug**
In case `ctas_approach=False` and `chunksize=True` the returned iterator raises:
```ValueError: 'chunksize' must be an integer >=1```

**To Reproduce**
```
dfs = wr.athena.read_sql_query(query_str, database='production', ctas_approach=False, chunksize=True)
for df in dfs:
    print(df.head())
```
awswrangler version = '1.10.0'
python version = '3.7.6'

Full Stack Trace:

```
~/.virtualenvs/ami-interactive/lib/python3.7/site-packages/awswrangler/athena/_read.py in _add_query_metadata_generator(dfs, query_metadata)
     55 ) -> Iterator[pd.DataFrame]:
     56     """Add Query Execution metadata to every DF in iterator."""
---> 57     for df in dfs:
     58         df = _apply_query_metadata(df=df, query_metadata=query_metadata)
     59         yield df

~/.virtualenvs/ami-interactive/lib/python3.7/site-packages/awswrangler/athena/_read.py in _fix_csv_types_generator(dfs, parse_dates, binaries)
     47 ) -> Iterator[pd.DataFrame]:
     48     """Apply data types cast to a Pandas DataFrames Generator."""
---> 49     for df in dfs:
     50         yield _fix_csv_types(df=df, parse_dates=parse_dates, binaries=binaries)
     51 

~/.virtualenvs/ami-interactive/lib/python3.7/site-packages/awswrangler/s3/_read_text.py in _read_text_chunked(paths, chunksize, parser_func, path_root, boto3_session, pandas_kwargs, s3_additional_kwargs, dataset, use_threads)
     58             boto3_session=boto3_session,
     59         ) as f:
---> 60             reader: pandas.io.parsers.TextFileReader = parser_func(f, chunksize=chunksize, **pandas_kwargs)
     61             for df in reader:
     62                 yield _apply_partitions(df=df, dataset=dataset, path=path, path_root=path_root)

~/.virtualenvs/ami-interactive/lib/python3.7/site-packages/pandas/io/parsers.py in read_csv(filepath_or_buffer, sep, delimiter, header, names, index_col, usecols, squeeze, prefix, mangle_dupe_cols, dtype, engine, converters, true_values, false_values, skipinitialspace, skiprows, skipfooter, nrows, na_values, keep_default_na, na_filter, verbose, skip_blank_lines, parse_dates, infer_datetime_format, keep_date_col, date_parser, dayfirst, cache_dates, iterator, chunksize, compression, thousands, decimal, lineterminator, quotechar, quoting, doublequote, escapechar, comment, encoding, dialect, error_bad_lines, warn_bad_lines, delim_whitespace, low_memory, memory_map, float_precision)
    686     )
    687 
--> 688     return _read(filepath_or_buffer, kwds)
    689 
    690 

~/.virtualenvs/ami-interactive/lib/python3.7/site-packages/pandas/io/parsers.py in _read(filepath_or_buffer, kwds)
    445     # Extract some of the arguments (pass chunksize on).
    446     iterator = kwds.get("iterator", False)
--> 447     chunksize = _validate_integer("chunksize", kwds.get("chunksize", None), 1)
    448     nrows = kwds.get("nrows", None)
    449 

~/.virtualenvs/ami-interactive/lib/python3.7/site-packages/pandas/io/parsers.py in _validate_integer(name, val, min_val)
    391             val = int(val)
    392         elif not (is_integer(val) and val >= min_val):
--> 393             raise ValueError(msg)
    394 
    395     return val

ValueError: 'chunksize' must be an integer >=1
```

**Comment by igorborgest** [comment]
Hi @czagoni, thanks for reaching out!

Bug fixed in the commit above. Be aware that with `ctas_approach=False` Athena produces only 1 output file, so you will receive a iterator with only 1 chunk. If you want smaller chunks, pass the number of lines you desire for each chunk instead of the `True` value.

Do you mind to install it directly from our repository and check if is everything fine for your use case before the official release?
The idea is to publish it in the version `1.10.1` next Wednesday.

```sh
pip install git+https://github.com/awslabs/aws-data-wrangler.git --no-use-pep517
```

**Comment by czagoni** [comment]
Hi @igorborgest , thanks for the quick fix!

I can confirm that the fix is working for me. 

Also, thanks for the heads up on the 1 chunk for `ctas_approach=False` - I wasn't aware of this. Is it worth putting this in the docs? (which also has `chunksize` and `chunked` as a param name at the mo)


```Note

chunksize argument (Memory Friendly) (i.e batching):

Enable the function to return an Iterable of DataFrames instead of a regular DataFrame.

There are two batching strategies on Wrangler:

If **chunksize=True**, a new DataFrame will be returned for each file in the query result.

If **chunked=INTEGER**, Wrangler will iterate on the data by number of rows igual the received INTEGER.

P.S. chunksize=True if faster and uses less memory while chunksize=INTEGER is more precise in number of rows for each Dataframe.```

**Comment by igorborgest** [comment]
@czagoni good catch! The commit above will fix it!

**Comment by czagoni** [comment]
@igorborgest excellent stuff - thank you!!!

**Comment by igorborgest** [comment]
Released on version [1.10.1](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.10.1)
